src/store/ducks/goals.ts

```diff
@@ -1,14 +1,15 @@
 import { get, keyBy, values } from 'lodash';
 import { AnyAction, Store } from 'redux';
 import SeamlessImmutable from 'seamless-immutable';
+import { PlanActionCodes } from '../../configs/settings';
 import { FlexObject } from '../../helpers/utils';
 
 /** the reducer name */
 export const reducerName = 'goals';
 
 /** interface for goal Object */
 export interface Goal {
-  action_code: string;
+  action_code: typeof PlanActionCodes[number];
```

Comment:
I think this one is already defined here https://github.com/onaio/reveal-frontend/blob/b0ea6c03e5479537a782aa5340b757087d3b5214/src/containers/forms/PlanForm/types.ts#L29

Reply:
oh cool, will switch